Applications/ApplicationsLib/ProjectData.cpp

```diff
@@ -448,19 +449,34 @@ void ProjectData::parseMedia(
          media_config->getConfigSubtreeList("medium"))
     {
         //! \ogs_file_attr{prj__media__medium__id}
-        auto const material_id = medium_config.getConfigAttribute<int>("id", 0);
+        auto const material_id_string =
+            medium_config.getConfigAttribute<std::string>("id", "0");
+        std::vector<std::string> material_ids;
+        boost::split(material_ids, material_id_string, boost::is_any_of(","));
 
-        if (_media.find(material_id) != _media.end())
+        for (auto const& m_id : material_ids)
```

Comment:
Not sure here: In the for-loop, the `m_id`, doesn't it need white space stripping?

Reply:
At this state, if the string contains a whitespace it would cause the OGS_FATAL.
I can simply remove all whitespaces from the entire string before the loop.